### DISCUSSION TITLE
post/react-query-2/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#2] [Announcements]
# post/react-query-2/

카카오페이에서 React Query를 활용하여 Concurrent UI Pattern을 도입한 사례에 대해 소개합니다. 이 글은 연작 중 2편에 해당합니다. 1편: 카카오페이 프론트엔드 개발자들이 React Query를 선택한 이유, 2편: React Query와 함께 Concurrent UI Pattern을 도입하는 방법

https://tech.kakaopay.com/post/react-query-2/

**Comment by thiporia** [comment]
감사히 잘 읽었습니다.

**Comment by taeyoungs** [comment]
잘 읽었습니다. 좋은 글 감사합니다!

**Comment by KirschX** [comment]
갑사합니다!!! 너무 유익하네요

**Comment by kimpra2989** [comment]
에러바운더리와 서스펜스를 사용하면 기존의 문제를 어떻게 개선할 수 있을지 명확히 알 수 있는 좋은 글이네요. 잘 읽었습니다. 감사합니다.
